src/thunderbots/software/launch/ai.launch

```diff
@@ -4,19 +4,24 @@
         Runs all of the AI nodes except for the backend (Grsim, MRF, etc.)
     -->
 
+    <param name="vision_multicast_address" value="$(optenv VISION_MULTICAST_ADDRESS 224.5.23.2)"/>
```

Comment:
When I try running `optenv VISION_MULTICAST_ADDRESS 224.5.23.2` locally I get `optenv: command not found`?

Reply:
When you say locally do you mean running it in terminal? Looks like optenv is a ROS specific thing. http://wiki.ros.org/roslaunch/XML

Reply:
Wow yup ok nvm. All good.